### PR TITLE
Add ftn data import

### PR DESCRIPTION
**Pull request by pseminatore**
Adding in import_ftn_data() function to import new FTN charting data from nflverse-data repo.

### CODE REVIEW

File: nfl_data_py/__init__.py [hunk 1]
```diff
@@ -1000,6 +1001,75 @@ def import_snap_counts(years):
...
+    if thread_requests:
+        with ThreadPoolExecutor() as executor:
+            # Create a list of the same size as years, initialized with None
+            data = [None]*len(years)
+            # Create a mapping of futures to their corresponding index in the data
+            futures_map = {
+                executor.submit(
+                    pandas.read_parquet,
+                    path=url.format(year),
+                    columns=columns if columns else None,
```
Comment (alecglen) [inline nfl_data_py/__init__.py:1050]: Simplified with removal of extra logic above.
```suggestion
                    columns=columns,
```

File: nfl_data_py/__init__.py [hunk 2]
```diff
@@ -1000,6 +1001,75 @@ def import_snap_counts(years):
...
+            }
+            for future in as_completed(futures_map):
+                data[futures_map[future]] = future.result()
+            data = pandas.concat(data)
+    else:
+        # read charting data
+        data = pandas.concat([pandas.read_parquet(url.format(x), engine='auto') for x in years])        
+
+    if columns:
+        data = data[columns]
```
Comment (alecglen) [inline nfl_data_py/__init__.py:1063]: Also unnecessary, the columns will be filtered by `read_parquet` columns parameter.
```suggestion
```

**Review by alecglen — COMMENTED** [review]
Appreciate the PR! All looks good, I just had a few minor suggestions for improvement.

The last thing it needs is a test case. It can pretty much be a copy of the `test_pbp` class. Let me know if you have questions on writing or running the test suite.

**Comment by pseminatore** [comment]
Appreciate the feedback! These changes have been implemented.

**Review by alecglen — APPROVED** [review]

**Pull request merged**